editor/src/core/model/element-metadata-utils.ts

```diff
@@ -2318,6 +2318,15 @@ export const MetadataUtils = {
     const element = MetadataUtils.findElementByElementPath(metadata, target)
     return MetadataUtils.isJSXMapExpressionFromMetadata(element)
   },
+  isJSXElementFromMetadata(element: ElementInstanceMetadata | null): boolean {
```

Comment:
I'm surprised these two don't already exist, though we do have `getJSXElementFromMetadata` and `getJSXElementFromElementInstanceMetadata`

Reply:
yeah, I followed the `isConditionalFromMetadata`, `isJSXMapExpressionFromMetadata`, etc pattern here - should I leave it like this (for completion and for the next developer who will need it and look for it) or do you think I need to use the `getJSXElement...` methods?

Reply:
so maybe we should leave this method for completion and for next developers, if that makes sense

Reply:
Yeah, keep the function 👍